README.md

```diff
@@ -199,28 +199,31 @@ To examine the current default configuration, run the command
-| `tcp-n2n`         | `checkTCPPort`  | TCP connection check from all pods of the daemon set of the host network to the node port used by the NWPD agent on the host network.                                 |
-| `tcp-n2p`         | `checkTCPPort`  | TCP connection check from all pods of the daemon set of the host network to pod endpoints (pod IP, port of GRPC server) of the daemon set running in the pod network. |
-
-The job IDs of the default configuration on the host (=node) network are using the naming convention `<jobtype-shortcut>-n[2<destination>][-(int|ext)]`.
+| Job ID            | Job Type        | Description                                                                                                                                                                                 |
+|-------------------|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `https-n2api-ext` | `checkHTTPSGet` | HTTPS Get check from all pods of the daemon set of the host network to the external address of the Kube API server.                                                                         |
+| `nslookup-n`      | `nslookup`      | DNS Lookup of IP addresses for the domain name `europe-docker.pkg.dev`, and external name of Kube API server.                                                                               |
+| `tcp-n2api-ext`   | `checkTCPPort`  | TCP connection check from all pods of the daemon set of the host network to the external address of the Kube API server.                                                                    |
+| `tcp-n2api-int`   | `checkTCPPort`  | TCP connection check from all pods of the daemon set of the host network to the internal address of the Kube API server.                                                                    |
+| `tcp-n2n`         | `checkTCPPort`  | TCP connection check from all pods of the daemon set of the host network to the node port used by the NWPD agent on the host network.                                                       |
+| `tcp-n2p`         | `checkTCPPort`  | TCP connection check from all pods of the daemon set of the host network to pod endpoints (pod IP, port of GRPC server) of the daemon set running in the pod network.                       |
+| `tcp-n2n-ipv6`    | `checkTCPPort`  | TCP connection check from all pods of the daemon set of the host network to the node port used by the NWPD agent on the host network using the IPv6 address of the node.                    |
+| `tcp-n2p-ipv6`    | `checkTCPPort`  | TCP connection check from all pods of the daemon set of the host network to pod IPv6 endpoints (IPv6 address of the pod, port of GRPC server) of the daemon set running in the pod network. |
```

Comment:
From a monitoring perspective this looks good to me. I don't see a massive problem with cardinality (factor of 2 at most) and the number of values that the `jobid` label can take are still limited. I have a question about the naming of the job IDs though: Is there a specific reason not to rename the `tcp-n2n` and `tcp-n2p` job IDs to `tcp-n2n-ipv4` and `tcp-n2p-ipv4`?

Reply:
There is actually not a technical reason. I didn't want to change the name for the majority of single-stack IPv4 clusters.